apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/metadata.json

```diff
@@ -22,9 +22,9 @@
       "type" : "Claim",
       "profile" : "http://hl7.org/fhir/StructureDefinition/Claim",
       "interaction" : [ {
-        "code" : "search-type"
-      }, {
         "code" : "read"
+      }, {
+        "code" : "search-type"
```

Comment:
I was curious about this re-ordering (here and elsewhere). If it's necessary, which library receiving an update requires this?

Reply:
Im not entirely sure. I just know the metadata.json had updates, and I ran the IT

Reply:
This would have changed due to a change in HAPI -- the HAPI library constructs this response for us based on the capabilities that we have configured the FHIR server with. They probably moved to a different data structure that preserves order (my guess). It can be tracked down in their release notes or their commit history but for an inconsequential change it is only necessary to do that to satisfy curiosity IMO (we have had more substantial changes from them in the last update that did require a changelog note).